docs/tutorials/text_prediction/automm.md

```diff
@@ -1,16 +1,19 @@
-# AutoMM for Image, Text, and Tabular
+# AutoMMPredictor for Image, Text, and Tabular
 :label:`sec_automm_predictor`
 
-AutoMM is a flexible framework that can train neural networks on image, text, numerical, and categorical data. For either single modal or multimodal data, AutoMM supports both classification and regression tasks. This tutorial demonstrates AutoMM's several application scenarios:
+Are you tired of switching codebases or hacking code for different data modalities (image, text, numerical, and categorical data) and tasks (classification, regression, and more)? `AutoMMPredictor` provides a one-for-all solution. This tutorial demonstrates several application scenarios.
```

Comment:
Consider to change `one-for-all solution` to `one-stop shop for multimodal deep learning models`

Reply:
Sure.